qcodes/instrument_drivers/signal_hound/USB_SA124B.py

```diff
@@ -178,6 +178,40 @@ def __init__(self, name, dll_path=None, **kwargs):
         self._trace_updated = False
         log.info('Initializing instrument SignalHound USB 124B')
         self.dll = ct.CDLL(dll_path or self.dll_path)
+
+        self.dll.saConfigCenterSpan.argtypes = [ct.c_int,
```

Comment:
does it make sense to extract these into a separate class (or smth) so that they don't pollute the driver constructor?

Reply:
Done